### PR TITLE
Domain transfer: Update google domain transfer helper copy

### PR DESCRIPTION
**Pull request by paulopmt1**
Related to 30-gh-Automattic/dotcom-marketing

## Proposed Changes

* Updated text of step 2 to be more assertive on google

## Testing Instructions

* Open http://calypso.localhost:3000/setup/google-transfer/intro and validate the new text

image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/components/google-domains-transfer-instructions/index.tsx [hunk 1]
```diff
@@ -16,6 +17,11 @@ const GoogleDomainsModal: React.FC< Props > = ( { children, className, focusedSt
 	const [ isOpen, setOpen ] = useState( false );
 	const openModal = () => setOpen( true );
 	const closeModal = () => setOpen( false );
+	const step2Text = useHasEnTranslation()(
+		'Select the domain you want to transfer in the "My domains" section.'
+	)
+		? __( 'Select the domain you want to transfer in the "My domains" section.' )
+		: __( "Click on the name of the domain that you'd like to transfer." );
```
Comment (ebinnion) [inline client/landing/stepper/declarative-flow/internals/components/google-domains-transfer-instructions/index.tsx]: Should we add, 'in the "My domains" section.'

Comment (paulopmt1) [reply client/landing/stepper/declarative-flow/internals/components/google-domains-transfer-instructions/index.tsx]: Yep, it sounds better @ebinnion, updated!

**Review by DustyReagan — APPROVED** [review]
Looks good to me!

Non-blocking: I might suggest a slight phrasing tweak from "name of the domain" to "domain name" and the inclusion of that "Note" in parenthesis Seth wrote [here](https://github.com/Automattic/dotcom-marketing/issues/30#issue-1831643852).  

> Click the domain name you want to transfer in the "My domains" section. (Note: You must unlock each domain individually.)

**Review by lupus2k — APPROVED** [review]
LGTM, and Spanish show the old copy because of no-translation yet 🚀

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8678877

Thank you @paulopmt1 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8678877).

**Pull request merged**